### PR TITLE
API: Changed package name from pyrisk to qrisk

### PR DESCRIPTION
**Pull request by ahgnaw**
Upgraded to latest versioneer and changed package name to qrisk.

### CODE REVIEW

**Comment by abhijeetkalyan** [comment]
@richafrank could you take a look at this?

**Comment by jdricklefs** [comment]
Any reason not to move this to `quantopian/qrisk`? I don't think it's widely used (yet)

**Comment by ahgnaw** [comment]
@jdricklefs I will rename the repo as soon as the package is renamed.

**Comment by richafrank** [comment]
> Any reason not to move this to quantopian/qrisk? I don't think it's widely used (yet)

Agreed - we'll change it.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[metadata]
+description-file = README.md
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = qrisk/_version.py
+versionfile_build = qrisk/_version.py
+tag_prefix = ""
+parentdir_prefix = qrisk-
```
Comment (richafrank) [inline setup.cfg]: Let's include a newline at the end here.

**Comment by richafrank** [comment]
LGTM!  And I changed the repo name.

**Pull request merged**